packages/fxa-admin-panel/interfaces/index.ts

```diff
@@ -22,6 +22,14 @@ export interface IClientConfig {
   servers: {
     admin: IServerInfo;
   };
+  sentry: {
+    dsn: string;
+    env: string;
+    serverName: string;
+    clientName: string;
+    sampleRate: number;
+  };
```

Comment:
Am a little surprised that we don't have a type for this elsewhere in the repo already.

Reply:
There is a type in fxa-shared/sentry/models/SentryConfigOpts.ts but it's mean to be applied at the root of the config. Perhaps this could be updated and repurposed.

Reply:
I actually just tried doing `Pick<SentryConfigOpts, "sentry">` here and I get a type error around `dsn`... probably doing something obviously wrong, but I think we could maybe yeah, repurpose it, and/or check to see if we can use this in other packages too.

![image](https://user-images.githubusercontent.com/13018240/190207902-1a89e151-23e1-4fa1-abc6-455a4ead2985.png)